From 64f5ae408c1a8f45e18946bb98bf1a01a23d88fd Mon Sep 17 00:00:00 2001
From: Laura Abbott <labbott@redhat.com>
Date: Mon, 19 Feb 2018 17:43:30 -0800
Subject: [PATCH] Comment out print triggering gcc-8 warning

Signed-off-by: Laura Abbott <labbott@redhat.com>
---
 tools/lib/bpf/libbpf.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/tools/lib/bpf/libbpf.c b/tools/lib/bpf/libbpf.c
index 97073d649c1a..121ad79d0803 100644
--- a/tools/lib/bpf/libbpf.c
+++ b/tools/lib/bpf/libbpf.c
@@ -1063,8 +1063,8 @@ bpf_program__reloc_text(struct bpf_program *prog, struct bpf_object *obj,
 	}
 	insn = &prog->insns[relo->insn_idx];
 	insn->imm += prog->main_prog_cnt - relo->insn_idx;
-	pr_debug("added %zd insn from %s to prog %s\n",
-		 text->insns_cnt, text->section_name, prog->section_name);
+	//pr_debug("added %zd insn from %s to prog %s\n",
+	//	 text->insns_cnt, text->section_name, prog->section_name);
 	return 0;
 }
 
-- 
2.14.3

